app/broadcast_areas/repo.py

```diff
@@ -127,13 +127,10 @@ def get_libraries(self):
         return sorted(libraries)
 
     def get_areas(self, area_ids):
-        q = """
-        SELECT id, name, count_of_phones, broadcast_area_library_id
-        FROM broadcast_areas
-        WHERE id IN ({})
-        """.format(",".join("?" * len(area_ids)))
+        q = "SELECT id, name, count_of_phones, broadcast_area_library_id FROM broadcast_areas"
+        where = "WHERE id IN ({})".format(",".join("?" * len(area_ids)))
```

Comment:
[python newbie question] I don't love this. Do we really need to include a `?` for each id that we want to insert into the statement?

Reply:
This seems to be the [preferred param substitution method](https://docs.python.org/2/library/sqlite3.html#cursor-objects) for lists for the sqlite3 driver.

Reply:
Yeah, that's what I gathered. It makes sense but I prefer other libraries where you can have `... WHERE id IN (?)` and then the library can intelligently sub in a list of ids into the single `?` so I wanted to be sure that wan't possible here either.

Reply:
sqlite3 has an executeMany which you could potentially sub for execute, but you'd need to refactor a bit to use it

Reply:
awesome, thank you. Not going to update this code, but I'll keep that in mind for future use.